params/variants.toml

```diff
@@ -4772,72 +4772,84 @@ rank = 1
 description = "Greek lower Iota (`ι`) like a straight line"
 selector."grek/iota" = "serifless"
 selector."grek/iota/sansSerif" = "serifless"
+selector."grek/iota/tailed" = "flatTailed"
```

Comment:
Could we just name it `selector."latn/iota"`? Maybe we could add a (currently identical) selector for `cyrl/iota`?

Reply:
I don't think having two selectors is that important. I mainly just wanted to match the name of the tau selector, not create a neutral selector between latin and cyrillic.